packages/kbn-apm-synthtrace/src/lib/apm/aggregators/create_service_summary_metrics_aggregator.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { ApmFields, hashKeysOf } from '@kbn/apm-synthtrace-client';
+import { identity, noop, pick } from 'lodash';
+import { createApmMetricAggregator } from './create_apm_metric_aggregator';
+
+const KEY_FIELDS: Array<keyof ApmFields> = [
+  'agent.name',
+  'service.environment',
+  'service.name',
+  'service.language.name',
+];
+
+export function createServiceSummaryMetricsAggregator(flushInterval: string) {
+  return createApmMetricAggregator(
+    {
+      filter: (event) =>
+        event['processor.event'] === 'metric' || event['processor.event'] === 'error',
```

Comment:
Shouldn't transactions also be included here? If only traces are ingested the service summary should still be produced, right?

Reply:
Yes, you're right - eventually I think we'd like to produce them only if there are no transactions, but that's not what APM Server does today (and also not what this aggregator does).